scripts/import_gilda_mappings.py

```diff
@@ -0,0 +1,41 @@
+import re
```

Comment:
maybe it makes sense to include this in the package hierarchy

Reply:
I suppose it could be but the scripts could have additional dependencies in general (we might want to use INDRA or PyOBO in some cases later) and aren't necessarily meant to be reusable so I'm not sure.

Reply:
I could go either way. If the code is in the package, then it might be easier to reuse with the `python -m ...`, but this is effectively the same in the end since nobody will be importing these